spec/requests/v1/developers_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+require 'spec_helper'
+require 'support/shared_context_for_api.rb'
+
+describe 'Api::V1::DevelopersController' do
+  include_context 'api'
+
+  describe 'GET /api/v1/developers/:developer_id' do
+    before do
+      @developer = FactoryBot.create(:developer,
```

Comment:
You don't need to write `FactoryBot` here because you created `spec/support/factory_bot.rb`.